03/app.js

```diff
@@ -1,3 +1,14 @@
 const x = 10;
 let iteration = 0;
-let randomNumber = -1;
\ No newline at end of file
+let randomNumber = -1;
+
+while(randomNumber !== x) {
+    randomNumber = Math.round(Math.random() * 20);
+    iteration = 1;
+    iteration += iteration;
+    console.log(randomNumber)
+    
+    if(randomNumber === x) {
+        console.log('Trafiłaś za ' + iteration + ' razem.')
+    }
```

Comment:
Można też zrezygnować z tego warunku, a sam komunikat dać poza pętlą.
Wtedy jak ona się zakończy to wiemy, że wylosowaliśmy odpowiednią liczbę ;)